api/provider/aws/load_balancer_create.go

```diff
@@ -0,0 +1,278 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (l *LoadBalancerProvider) Create(req models.CreateLoadBalancerRequest) (*models.LoadBalancer, error) {
+	loadBalancerID := generateEntityID(req.LoadBalancerName)
+	fqLoadBalancerID := addLayer0Prefix(l.Config.Instance(), loadBalancerID)
+	fqEnvironmentID := addLayer0Prefix(l.Config.Instance(), req.EnvironmentID)
+
+	environmentSGName := fmt.Sprintf("%s-env", fqEnvironmentID)
```

Comment:
Yeah I'm a little torn on this. Reason being it takes a step closer to our [current ID implementation](https://github.com/quintilesims/layer0/blob/develop/api/backend/ecs/id/id.go#L107). I don't really feel too strongly either way - only that if we use a common helper function we are very vigilant about keeping it simple/sane.